cookbooks/fb_init_sample/recipes/default.rb

```diff
@@ -79,7 +79,9 @@
 include_recipe 'fb_hostconf'
 include_recipe 'fb_sysctl'
 # HERE: networking
-include_recipe 'fb_users'
```

Comment:
Looks like this got pulled in by mistake

Reply:
not a mistake. As I said in the PR description:
```
This is based on top of https://github.com/facebook/chef-cookbooks/pull/255 - and together Debian kitchen tests
now pass.
```

I can't test this without that as it never gets there. Once you merge the users one I"ll rebase all these